<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Election Night Data Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showData1Button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="statusArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(250, 255, 137);
color: #000000</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county1Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Lake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download1Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county1Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://results.enr.clarityelections.com/IL/Lake/114135/300541/reports/detailxml.zip</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county2Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>DuPage</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county3Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Will</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county4Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>McHenry</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county4Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://results.enr.clarityelections.com//IL/McHenry/115186/301334/reports/detailxml.zip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county3Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://results.enr.clarityelections.com//IL/Will/114217/300664/reports/detailxml.zip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county2Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://www.dupageresults.gov//IL/DuPage/114213/300666/reports/detailxml.zip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download2Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download3Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download4Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData2Button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData3Button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData4Button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county5Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Cook</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county6Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Kane</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county7Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Boone</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="county8Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>DeKalb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county5Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://results622.cookcountyclerkil.gov/Home/Detail?contestId=203</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county6Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>http://electionresults.countyofkane.org/Contests.aspx?Id=26</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county7Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>https://www.boonecountyil.gov/Departments/Clerk-Recorder/voting/2022_jun_28_il_boone_SOVC.pdf</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="county8Url">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>http://dekalb.il.clerkserve.com/wp-content/uploads/DeKalb-General-Primary-Election-Official-Results-06-28-2022-1.pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download5Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download6Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download7Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download8Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData5Button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>200</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData6Button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData7Button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>290</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData8Button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
